docs/import/add_disaster.js

```diff
@@ -370,112 +376,76 @@ function populateStateAssetPickers() {
-    row.append(createRadioButtonCell(name, 'poverty'));
-    row.append(createRadioButtonCell(name, 'svi'));
-    row.append(createRadioButtonCell(name, 'income'));
-    row.append(createRadioButtonCell(name, 'none', true));
-    row.append($(document.createElement('td'))
-                   .append($(document.createElement('div')).attr({
-                     id: name + '-columns',
-                   })));
-    $('#asset-selection-table-body').append(row);
-  }
-}
-
-/**
- * Handles the user selecting a type for one of the possible score assets.
- * @param {String} asset The asset path
- * @param {String} name The name of the row
- */
-function handleScoreAssetSelection(asset, name) {
-  const selectedType = $('input[name=\'' + name + '-select\']:checked').val();
-  const columnCell = $('#' + name + '-columns');
-  columnCell.empty();
-  if (selectedType != 'none') {
-    columnCell.append($(document.createElement('input')).attr({
-      id: name + '-column',
-      type: 'text',
-    }));
-    if (selectedType == 'poverty') {
+    row.append(createTd().append(
+        $(document.createElement('div')).html(scoreAssetType)));
+    for (const state of states) {
+      if (stateAssets.get(state)) {
+        row.append(createTd().append(createAssetDropdown(
+            stateAssets.get(state), scoreAssetType, state)));
+      }
+    }
+    // TODO: Make column names selects instead.
+    const columnCell = createTd();
+    for (let i = 0; i < scoreAssetTypes[scoreAssetType].length; i++) {
       columnCell.append($(document.createElement('input')).attr({
-        id: name + '-column-2',
+        id: scoreAssetType + '-column-' + i,
         type: 'text',
       }));
     }
+    row.append(columnCell);
+    tableBody.append(row);
+    row.change(() => handleScoreAssetSelection(scoreAssetType));
   }
-  // TODO: Write the asset name and type to firebase here?
-  // TODO: Add ability to write column names to firebse and make column names
-  // selects instead.
 }
 
 /**
- * Creates a radio button for the select score data table.
- * @param {String} name The name of the row/base value for radio button id, etc.
- * @param {String} value The value of the radio button
- * @param {?boolean} opt_checked Whether to check the radio button
+ * Initializes a dropdown with assets.
+ * @param {Array<string>} assets array of assets for add to dropdown
+ * @param {string} row The asset type/row to put the dropdown in.
+ * @param {string} state The state the assets are in.
  */
-function createRadioButtonCell(name, value, opt_checked) {
-  return $(document.createElement('td'))
-      .append($(document.createElement('input')).attr({
-        id: name + '-' + value,
-        type: 'radio',
-        name: name + '-select',
-        value: value,
-        checked: opt_checked
-      }));
+function createAssetDropdown(assets, row, state) {
+  // Create the asset selector and add a 'None' option.
+  const select = $(document.createElement('select')).attr({
```

Comment:
can be slightly more concise with .prop('id', row + '-' + state) instead of the whole attr shebang